src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -69,11 +69,18 @@
       [rn/view {:style style/space-between-suboptions}]
       [quo/small-option-card
        {:variant  :icon
-        :title    (i18n/label :t/use-recovery-phrase)
+        :title    "Temporary (old) recover phrase flow"
```

Comment:
move to translations

Reply:
I think it's ok to hardcode this, so that once we remove it we don't have to remove the translation in the json file, what do you think?